wiki/update_wiki.py

```diff
@@ -103,12 +109,14 @@ def generate_home_wiki_page(wiki_dir: Path, repository: str) -> None:
 def update_monthly_wiki_page(
     wiki_dir: Path, year_month: str, build_history_line: str
 ) -> None:
-    MONTHLY_PAGE_HEADER = f"""\
-# Images built during {year_month}
+    MONTHLY_PAGE_HEADER = textwrap.dedent(
+        f"""\
+        # Images built during {year_month}
```

Comment:
This table didn't and won't look nice, because we're appending lines to the existing file and the length is variable